test/functional/job_invocations_controller_test.rb

```diff
@@ -46,4 +46,16 @@ class JobInvocationsControllerTest < ActionController::TestCase
     assert_equal(template_invocation_params,
       assigns(:composer).params['template_invocations'])
   end
+
+  test 'new via GET and POST' do
+    template = FactoryBot.create(:job_template, :with_input)
+    feature = FactoryBot.create(:remote_execution_feature, job_template: template)
+    params = { feature: feature.label, inputs: { template.template_inputs.first.name => 'foobar' } }
+
+    get :new, params: params, session: set_session_user
+    assert_response :success
+
+    post :new, params: params, session: set_session_user
```

Comment:
I tried putting a pry into the controller to see what happens

The get request:
```
> request.path
=> "/job_invocations/new"
> request.query_string
=> "feature=remote_execution_feature_3&inputs%5BTemplate+input+2%5D=foobar"
> request.body.read
=> ""
```

The post request
```
> request.path                                                                                                             
=> "/job_invocations/new"
> request.query_string
=> ""
> request.body.read                                                                                                        
=> "feature=remote_execution_feature_3&inputs%5BTemplate+input+2%5D=foobar"
```

It looks like what was query string in get, gets sent as body of the request when doing a post. Is this expected?

Reply:
Yeah, that is expected, since we need to send remediation ids in body of request, not in URL. @xprazak2 - on UI, when you build the Remediation plan, you will send it as form POST right?

Reply:
> we need to send remediation ids in body of request, not in URL.

I'm aware of that. I was more concerned about the format of the body. Shouldn't it be json?

Reply:
@xprazak2 Are you going to send data by submitting form or as json request?

Reply:
Not Ondra, but it will be sent over as form data

Reply:
In that case it works correctly.